src/redux.js

```diff
@@ -443,12 +421,6 @@ export default function rootReducer(state = initialState, action) {
       trainedModel: action.trainedModel
     };
   }
-  if (action.type === SET_TRAINED_MODEL_DETAILS) {
-    return {
-      ...state,
-      trainedModelDetails: action.trainedModelDetails
-    };
-  }
   if (action.type === SET_TRAINED_MODEL_DETAIL) {
```

Comment:
Just clarifying, `setTrainedModelDetails` and `setSelectedFeatures` can both be removed because we edit the individual fields for those objects?

Reply:
It looks like we do still have actions that modify them, at https://github.com/code-dot-org/ml-playground/blob/0c27d7ee37550eda2b7fc11b7821cf217507bd1e/src/redux.js#L452 and https://github.com/code-dot-org/ml-playground/blob/0c27d7ee37550eda2b7fc11b7821cf217507bd1e/src/redux.js#L339-L356 respectively.